### ISSUE TITLE
Inaccuracy of inductance computation

### ISSUE THREAD

**Issue by pfuchs** [LUMC-LowFieldMRI/GradientDesignTool#3]
The inductance computation included in the tool is a guideline at best, and not at all dependable or stable. It would make sense to either change the algorithm (internally use a magnetostatic solver on the line segments or something along those lines) or investigate the inaccuracies of the current algorithm using a commercial solver and some test cases to improve this feature.

**Comment by ShannonZ** [comment]
Is it the reason causing large error (Error over 140mm DSV>50%) in X direction using default parameters?
![LargeError](https://user-images.githubusercontent.com/7865691/155088057-1b8170ba-7e14-4c37-b837-445871221ab4.png)

**Comment by pfuchs** [comment]
No, the inductance calculation is done separate from the field/gradient computation, so they should not lead to any error there.
